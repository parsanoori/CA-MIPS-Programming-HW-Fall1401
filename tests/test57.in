96
712965949
604364276
665889283
359116362
875542770
782750855
589848373
516529669
814577367
688616359
1035011775
483756417
27168917
18802138
45774665
612579957
86727336
432436644
1000045230
739288651
382295678
301172585
834331049
922012039
295427760
607484465
885485209
366321285
464154953
262356174
669110788
491667846
625439943
579513997
475762463
702760759
994012674
333559142
400311638
677183512
799665603
874227669
260355041
967025768
299999872
1021705899
789206338
837377288
1023691461
633530485
744813404
443269143
847938867
933584941
361733098
1003328552
179881309
649807121
522459559
107802547
1050343029
77682017
352367553
161377427
481121268
947975822
21133273
19454289
29109325
427871973
528242427
403630236
79731935
109470605
290166252
553437579
72931875
866094478
794924083
561251305
505021216
329326028
931004378
998738355
548641901
917000896
49945593
618371502
746768891
628345103
298951673
909265986
156455376
939995469
540544140
376099289