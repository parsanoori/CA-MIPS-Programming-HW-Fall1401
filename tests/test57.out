18802138
19454289
21133273
27168917
29109325
45774665
49945593
72931875
77682017
79731935
86727336
107802547
109470605
156455376
161377427
179881309
260355041
262356174
290166252
295427760
298951673
299999872
301172585
329326028
333559142
352367553
359116362
361733098
366321285
376099289
382295678
400311638
403630236
427871973
432436644
443269143
464154953
475762463
481121268
483756417
491667846
505021216
516529669
522459559
528242427
540544140
548641901
553437579
561251305
579513997
589848373
604364276
607484465
612579957
618371502
625439943
628345103
633530485
649807121
665889283
669110788
677183512
688616359
702760759
712965949
739288651
744813404
746768891
782750855
789206338
794924083
799665603
814577367
834331049
837377288
847938867
866094478
874227669
875542770
885485209
909265986
917000896
922012039
931004378
933584941
939995469
947975822
967025768
994012674
998738355
1000045230
1003328552
1021705899
1023691461
1035011775
1050343029